mods/d2k/rules/structures.yaml

```diff
@@ -49,9 +49,9 @@ construction_yard:
 	Selectable:
 		Bounds: 96,64
 	Health:
-		HP: 1000
+		HP: 3000
 	Armor:
-		Type: Concrete
+		Type: cy
```

Comment:
OK looking at the "Damage versus" section of the rules (in TibEd), that doesn't make any sense. Also TibEd does say that "CY" stands for "Construction Yard". So I guess it's quite possible the "LIGHT" value is an error.